evaluators/text/RougeEvaluator/__init__.py

```diff
@@ -0,0 +1,27 @@
+from typing import Sequence
+
+from rouge import Rouge
```

Comment:
Move external imports to inside the functions or at post init. Otherwise warning methd will appear at any jina application